subiquity/cmd/tui.py

```diff
@@ -35,12 +35,23 @@ def parse_options(argv):
     parser = argparse.ArgumentParser(
         description='SUbiquity - Ubiquity for Servers',
         prog='subiquity')
+    try:
+        ascii_default = os.ttyname(0) == "/dev/ttysclp0"
+    except OSError:
+        ascii_default = False
+    parser.set_defaults(ascii=ascii_default)
```

Comment:
Ooh I didn't know about this method. I suspect there is some other stuff that could be made simpler using it but I forget what now!